core/lib/misc.js

```diff
@@ -235,14 +235,30 @@ const teleportUserInLevel = (user, level, source = 'teleporter') => {
   return level.name;
 };
 
+const canUseLevelFeature = (user, featureName, showNotification = true) => {
```

Comment:
`showNotification` doesn't seem used :)

```suggestion
const canUseLevelFeature = (user, featureName) => {
```

Reply:
It was used for globalChat to not showing it since we have a message error in console, we can show it as notification too.